File: task3.py
#  Find the "Kth" min element of an array

def findkthMinimum(numbers, k):

    numbers.sort()
    print(numbers)
    return numbers[k-1]


numbers = [1, 43, 24, 124, 35]
k = 3
print(k, "Minimum number is ", findkthMinimum(numbers, k))

File: task3.js
// Find the "Kth" min element of an array 

function findKthSmallest(params, k) {
    return params.sort()[k - 1]

}



let numbers = new Array(5)
numbers = [87, 12, 32, 24, 35, 15, 43]
let k = 2

console.log(k, "th Smallest number is ", findKthSmallest(numbers, k))